studio/components/to-be-cleaned/SqlEditor/TabWelcome.js

```diff
@@ -1,72 +1,41 @@
 import React, { useState } from 'react'
 import { observer } from 'mobx-react-lite'
-import { useRouter } from 'next/router'
-import { IconChevronRight, IconLoader, Typography } from '@supabase/ui'
+import { IconChevronRight, Typography } from '@supabase/ui'
 
 import { useSqlStore } from 'localStores/sqlEditor/SqlEditorStore'
-import { useProjectContentStore } from 'stores/projectContentStore'
 
 import Telemetry from 'lib/telemetry'
-import { useStore } from 'hooks'
+import { useOptimisticSqlSnippetCreate } from 'hooks'
 import { partition } from 'lodash'
 
-import { createSqlSnippet } from './SqlEditor.utils'
 import { SQL_TEMPLATES } from 'components/interfaces/SQLEditor/SQLEditor.constants'
 import CardButton from 'components/ui/CardButton'
 
 const TabWelcome = observer(() => {
-  const router = useRouter()
-  const { ref } = router.query
-
-  const { ui } = useStore()
-  const { profile: user } = ui
-
-  const sqlEditorStore = useSqlStore()
-  const contentStore = useProjectContentStore(ref)
-
   const [SQL, QuickStart] = partition(SQL_TEMPLATES, { type: 'template' })
```

Comment:
can we avoid variables starting with uppercase as a syntax guideline?

should be reserved for classes

Reply:
Yep! That was like that before I got to it 😉
Happy to fix as part of this PR?